javascript/javascript1/week2/weather.js

```diff
@@ -0,0 +1,21 @@
+let clothes;
+let temp;
+function clothesToWear(temp)
```

Comment:
Rename `temp` to `temperature to make more clear and not thinking about it as temporary